tests/QAN/overview_test.js

```diff
@@ -9,6 +11,28 @@ Before(async ({
   qanFilters.waitForFiltersToLoad();
 });
 
+Scenario(
+  'PMM-T207 Verify hovering over query in overview table  @qan',
+  async ({ I, qanOverview }) => {
+    qanOverview.waitForOverviewLoaded();
+    const firstQueryLocator = `${qanOverview.getRowLocator(1)} div.td:nth-child(2) div`;
+
+    I.waitForVisible(`${firstQueryLocator} > div`, 30);
+    let firstQueryText = await I.grabTextFrom(`${firstQueryLocator} > div`);
+
+    firstQueryText = qanOverview.removeSymbolFromString(firstQueryText, ' ');
+
+    I.moveCursorTo(`${firstQueryLocator} > svg`);
+    I.waitForVisible(qanOverview.elements.tooltipQueryValue, 30);
+    I.seeElement(qanOverview.elements.tooltipQueryValue);
+    let tooltipQueryText = await I.grabTextFrom(qanOverview.elements.tooltipQueryValue);
+
+    tooltipQueryText = qanOverview.removeSymbolFromString(tooltipQueryText, ' ');
+    tooltipQueryText = qanOverview.removeSymbolFromString(tooltipQueryText, '\n');
```

Comment:
you could do the same in one line

Reply:
did you try string.replace function to achieve the same result?

Reply:
fixed. thanks